src/plots/plots.js

```diff
@@ -527,6 +563,9 @@ plots.supplyDataDefaults = function(dataIn, dataOut, layout) {
         basePlotModules = layout._basePlotModules = [],
         cnt = 0;
 
+    // Make this idempotent by emptying the existing array:
+    dataOut.length = 0;
```

Comment:
Oh. Is this an artifact, now? Looks like no new `Plots.supplyDataDefaults` calls were added in this PR.